src/cube.js

```diff
@@ -237,7 +237,13 @@ class Cube {
             for (let i = 0; i < storeSize; ++i) {
                 for (let j = 0; j < numMeasures; ++j) {
                     params[measureIds[j]] = measures[j].getValue(i);
-                    params[`${measureIds[j]}__total`] = measures[j].total;
+
+                    const formulaContainTotal = this.computedMeasures[measureId]
+                        .toString()
```

Comment:
i think this could be moved outside the double loop. `this.computedMeasures[measureId].toString()` will be the same for the entire cube, but currently it is evaluated `storeSize * numMeasures` times